pkg/asset/installconfig/vsphere/vsphere.go

```diff
@@ -170,6 +178,36 @@ func getClients() (*vCenterClient, error) {
 	}, nil
 }
 
+// checkComputeResource checks the ClusterComputeResource to see if there are any attached hosts.  Without
+// hosts, the check for networks will fail.  We want the message to be clear if this is the cause so
+// users know what to address.  This only returns an error if unable to verify or if no ComputeResources
+// are found.
+func checkComputeResource(ctx context.Context, datacenter string, cluster string, finder Finder) error {
+	ctx, cancel := context.WithTimeout(ctx, 60*time.Second)
+	defer cancel()
+
+	path := fmt.Sprintf("/%s/host/%s", datacenter, cluster)
+	ccr, err := finder.ClusterComputeResource(context.TODO(), path)
```

Comment:
You just created a child context above, shouldn't you use it here?

Reply:
Thanks, I 'll fix that.